### ISSUE TITLE
Invalid log message when stop postgreSql by pgSql RA

### ISSUE THREAD

**Issue by m-ly4** [ClusterLabs/resource-agents#692]
pgsql RA - pgsql_real_stop() function:
In log message value of ${OCF_RESKEY_stop_escalate} is used:
```bash
    if pgsql_status
    then
        #PostgreSQL is still up. Use another shutdown mode.
        ocf_log info "PostgreSQL failed to stop after ${OCF_RESKEY_stop_escalate}s using -m fast. Trying -m immediate..."
        runasowner "$OCF_RESKEY_pgctl -D $OCF_RESKEY_pgdata stop -m immediate"
    fi
```

But above initiated local variable stop_escalate, which can have another value for slave agent:
```bash
    stop_escalate=$OCF_RESKEY_stop_escalate
    if [ "$1" = "slave" ]; then
        stop_escalate="$OCF_RESKEY_stop_escalate_in_slave"
    fi
```
```

**Comment by kskmori** [comment]
Thanks for reporting.
I and my colleague will look into this issue.

**Comment by kskmori** [comment]
@m-ly4 : Please find that it has been fixed as the PR #736 . Thanks!
Closing.
